TopMovies/View/MovieCollectionViewCell.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  MovieCollectionViewCell.swift
+//  TopMovies
+//
+//  Created by anikolaenko on 08.02.2021.
+//
+
+import UIKit
+import SnapKit
+import Nuke
+
+class MovieCollectionViewCell: UICollectionViewCell {
+    private let posterImage = UIImageView()
+    private let adultLabel = UILabel()
+    private let ratingLabel = UILabel()
+    private let titleLabel = UILabel()
+    private let descriptionLabel = UILabel()
+    private let shortInfoStackView = UIStackView()
+    private let infoStackView = UIStackView()
+    
+    public func configureWith(movie: Movie) {
+        if let url = movie.poster { Nuke.loadImage(with: url, into: posterImage) }
+        adultLabel.text = "Adult: \(movie.adult ? "Yes" : "No")"
+        ratingLabel.text = String(movie.rating)
+        titleLabel.text = movie.title
+        descriptionLabel.text = movie.description
+    }
```

Comment:
Ok so Movie is a model object that is used for business logic and eventually it should be converted in some view representation outside of UI.

So instead let's create a struct named MovieCollectionProps. It won't just create an aditional layer between business logic and UI, but it will also allow you to descripe that UI using properies of that struct:

```swift
struct MovieCollectionProps {
  let adultLabelText: String
  let ratingLabelText: String
  let titleLabelText: String
  let descriptionLabeltext: String
  let posterURL: URL?
  let posterPlaceholderImage: UIImage 
  // in case posterURL is nil while there has to be something shown instead
}
```

If you have cases when some label should be hidden or it's text color should be different, you can also describe that in those props:

```swift
  //...
  let adultLabelTextColor: UIColor
  let isDescriptionLabelHidden: Bool
  //...